plotly/figure_factory/_facet_grid.py

```diff
@@ -189,18 +190,22 @@ def _facet_grid_color_categorical(df, x, y, facet_row, facet_col, color_name,
     if not facet_row and not facet_col:
         color_groups = list(df.groupby(color_name))
         for group in color_groups:
-            trace = graph_objs.Scatter(
+            trace = dict(
                 x=group[1][x],
                 y=group[1][y],
```

Comment:
You found an oversight in my design.

Originally this was meant for just scatter, so I was forcing `x` and `y`. I'm going to try not forcing `x` and `y` and putting `df[x]` and `df[y]` in the trace dictionary depending on what the user provides